resources/js/Pages/Home.vue

```diff
@@ -42,18 +43,51 @@
             user: Object,
         },
         methods: {
+            checkEmpty() {
+                if( !this.form.itemsInput ) {
+                    this.error = {
+                        type: 'warning',
+                        message: 'item-form-error-message-empty'
```

Comment:
To continue the first comment made, I think you might be able to use the i18n function here like so (though I didn't try it out):

```suggestion
                        message: this.$i18n('item-form-error-message-empty')
```